include/Version.h.in

```diff
@@ -13,6 +13,6 @@
 // See the License for the specific language governing permissions and limitations
 // under the License.
 //
-#define VERSION_MAJOR @gw_VERSION_MAJOR@
-#define VERSION_MINOR @gw_VERSION_MINOR@
-#define VERSION_PATCH @gw_VERSION_PATCH@
+#define VERSION_MAJOR @mw_VERSION_MAJOR@
```

Comment:
Will this cause issues when the string is not changed in `libflute`? See `libflute/include/Version.h.in`

Reply:
``Version.h.in`` in libflute should use its own version (@libflute_VERSION_MAJOR@ etc).

I'll change it there.